main.py

```diff
@@ -113,39 +110,9 @@ def main():
             pc_reserve_df["key"] = list(executor.map(obfuscate, pc_reserve_df["key"]))
 
         # Query Sierra for patron info using the patron barcodes
-        barcodes_str = "','".join(
-            pc_reserve_df["barcode"].to_string(index=False).split()
-        )
-        barcodes_str = "'" + barcodes_str + "'"
-        sierra_query = build_sierra_query(barcodes_str)
-
+        barcodes_str = "'b" + "','b".join(pc_reserve_df["barcode"].unique()) + "'"
         sierra_client.connect()
-        logger.info(f"Setting Sierra query timeout to {sierra_timeout} minutes")
-        sierra_client.execute_query(f"SET statement_timeout='{sierra_timeout}min';")
-        logger.info("Querying Sierra for patron information by barcode")
-
-        initial_log_level = logging.getLogger("postgresql_client").getEffectiveLevel()
-        logging.getLogger("postgresql_client").setLevel(logging.CRITICAL)
-        finished = False
-        num_attempts = 1
-        while not finished:
-            try:
-                sierra_raw_data = sierra_client.execute_query(sierra_query)
-                finished = True
-            except PostgreSQLClientError as e:
-                if num_attempts < max_sierra_attempts:
-                    logger.info("Query failed -- trying again")
-                    num_attempts += 1
-                else:
-                    logger.error(
-                        f"Error executing Sierra query {sierra_query}:" "\n{e}"
-                    )
-                    sierra_client.close_connection()
-                    s3_client.close()
-                    kinesis_client.close()
-                    raise e from None
-        logging.getLogger("postgresql_client").setLevel(initial_log_level)
-
```

Comment:
No it's part of [this one](https://newyorkpubliclibrary.atlassian.net/browse/DE-166). The reason we were originally retrying is because the query would time out and fail a bunch. With this new query we don't expect that to happen.